tests/memoryview/memslice.pyx

```diff
@@ -21,6 +21,12 @@ if sys.version_info[0] < 3:
 else:
     import builtins
 
+try:
+    from this_module_doesnt_exist import *
```

Comment:
Let's use something that we can at least control.
```suggestion
    from Cython.Tests.this_module_does_not_exist import *
```

Reply:
For what it's worth it doesn't matter if the import fails or not so we could test a successful import instead. I just thought that a failed one was less likely to introduce unwanted names.